Commands/Record/Presenter/RecordFormatter.cs

```diff
@@ -39,9 +39,21 @@ private string FormatRecordRanking(string username, CounterCategory category, lo
 
     private string FormatRecord(RecordEntity toFormat, bool shouldIncludeCategory)
     {
-        var reason = toFormat.Motive == null
-            ? "*For reasons unknown to History*"
-            : $"*« {toFormat.Motive} »*";
+        var reason = "";
+        if (toFormat.Category.Equals(CounterCategory.Raclette))
+        {
+            long MotiveLong;
+            long.TryParse(toFormat.Motive, out MotiveLong);
+            reason = toFormat.Motive == null
+                ? "*Unknown Raclette*"
+                : $"*« Raclette of {DateHelper.FromTimestampToDateTime(MotiveLong).ToString("dd-MM-yyyy")} »*";
```

Comment:
you could have used `DateHelper. FromDateTimeToStringDate` instead, which results in about the same